src/pages/index.tsx

```diff
@@ -295,15 +300,17 @@ export default function IndexPage() {
 
         {!loading && (
           <Row>
-            {ff.flags && ff.flags[FeatureFlags.FilterType] && (
+            {ff.flags && Object.values(ff.flags).find((flag) => flag) && 
```

Comment:
I'd prefer the other way and add each other filter when necessary

```suggestion
            {ff.flags && ff.flags[FeatureFlags.FilterType] &&
```